repository/MonticelloFileTree-Core.package/MCFileTreeRepository.class/instance/goferVersionFrom..st

```diff
@@ -1,10 +0,0 @@
-as yet unclassified
-goferVersionFrom: aVersionReference
-    "until we no longer find .tree directories in the wild"
-
-    ((self readableFileNames collect: [ :fileName | self fileDirectoryOn: fileName ])
-        select: [ :packageDirectory | self fileUtils directoryExists: packageDirectory ])
-        collect: [ :packageDirectory | 
-            (self versionInfoForPackageDirectory: packageDirectory) name = aVersionReference name
-                ifTrue: [ ^ self loadVersionFromFileNamed: (self fileUtils directoryName: packageDirectory) ] ].
-    ^ nil
```

Comment:
Hm. I don't recall removing this guy… in fact I still have it in the image, except it's an extension from `gofer-core`. I might be responsible for an automatic recategorization though…